check-osp/osp_checks.py

```diff
@@ -112,17 +112,24 @@ def check_services_status(system, **kwargs):
         # if a hostname contains localhost, we want to avoid trying to connect
         if 'localhost' in host:
             continue
-        ssh = ssh_client(host, username="root", password=system.password)
-        with ssh:
-            for service_name, status in services.iteritems():
-                service_status = is_service_in_status(ssh, service_name, status)
-                try:
-                    hosts_agents[host].update({service_name: service_status})
-                except KeyError:
-                    hosts_agents[host] = {service_name: service_status}
-
+        with ssh_client(host, username="root", password=system.password) as ssh:
+            service_status_dict = get_services_status_list(ssh)
+        for service_name, expected_status in services.items():
+            # if service_status_dict has service `service_name` get it status
+            # compare it with status
+            try:
+                logger.debug("service:{} status: {} expected_status: {}"
+                    .format(service_name, service_status_dict[service_name], expected_status))
+                service_status = (expected_status in service_status_dict[service_name])
+            except KeyError:
+                # This is because not all hosts may have all services installed
+                logger.debug("Service {} not found on host {}".format(service_name, host))
```

Comment:
Why `debug` level? I would expect it to be `Info`, what do you think?

Reply:
I thought it would fill up logs too quickly. Do you still want to make it Info?

Reply:
good point, debug is fine